data/map.txt

```diff
@@ -25778,6 +29286,135 @@ system Vega
 		distance 2981.84
 		period 1710.41
 
+system Vik-benoureal
+	pos 195.47178 -1.8426778
+	government Xapleaux
+	attributes exposed hot xapleaux
+	link Atke-babounriuer-rik
+	link Dygit-hourimaserai
+	link Dygit-jarobouillier-cot
+	link Ifk-groundissemour
+	asteroids "small rock" 9 1.152
+	asteroids "medium rock" 53 0.9
+	asteroids "large rock" 3 1.404
+	asteroids "small metal" 1 1.008
+	asteroids "medium metal" 4 1.584
+	asteroids "large metal" 1 0.72
+	minables titanium 2 1.83489
+	haze _menu/haze-fullbright
+	fleet "Xapleaux Small Cargo" 1200
+	fleet "Xapleaux Small Security" 1200
+	fleet "Xapleaux Small Passenger" 1200
+	habitable 486.68
+	belt 1606
+	trade Clothing 216
+	trade Electronics 683
+	trade Equipment 612
+	trade Food 297
+	trade "Heavy Metals" 976
+	trade Industrial 550
+	trade "Luxury Goods" 1222
+	trade Medical 648
+	trade Metal 338
+	trade Plastic 444
+	hazard "Core Exposed Heat" 1
+	hazard "Core Heat Flare" 1200
+	hazard "Core Shadow Passing" 21000
+	object
+		sprite star/k0
+		period 10
+	object
+		sprite planet/venus
+		distance 153.01
+		period 34.3177
+	object
+		sprite planet/miranda
+		distance 316.26
+		period 101.978
+	object
+		sprite planet/cloud2
+		distance 876.62
+		period 470.604
+	object
+		sprite planet/gas0
+		distance 1759.83
+		period 1338.58
+		object
+			sprite planet/desert4
+			distance 254
+			period 11.8633
+
+system Vikinik-bounera
+	pos 14.027703 -9.0649
+	government Xapleaux
+	attributes exposed hot xapleaux
+	link Atch-werimoural-nikt
+	link Crxst-xialourum
+	link Icibvirk-gwolav-jek
+	asteroids "large rock" 2 2.964
+	asteroids "small metal" 28 2.8652
+	asteroids "medium metal" 9 2.4206
+	asteroids "large metal" 2 3.3098
+	minables copper 1 5.54352
+	minables tungsten 9 5.10205
+	haze _menu/haze-fullbright
+	fleet "Xapleaux Small Cargo" 1200
+	fleet "Xapleaux Small Security" 1200
+	fleet "Xapleaux Small Passenger" 1200
+	fleet "Xapleaux Small Cargo" 1200
+	fleet "Xapleaux Small Security" 1200
+	fleet "Xapleaux Small Passenger" 1200
+	habitable 233.28
+	belt 1353
+	trade Clothing 271
+	trade Electronics 812
+	trade Equipment 687
+	trade Food 259
+	trade "Heavy Metals" 650
+	trade Industrial 795
+	trade "Luxury Goods" 1164
+	trade Medical 685
+	trade Metal 280
+	trade Plastic 443
+	hazard "Core Exposed Heat" 1
+	hazard "Core Heat Flare" 1200
+	hazard "Core Shadow Passing" 21000
+	object
+		sprite star/m4
```

Comment:
The M4 is kept only for backwards compatibility, it's intended to be removed in the future.  It has no high-dpi sprite IIRC and is out of pattern with the other stars.  Probably not a good idea to add them.

```suggestion
		sprite star/m3
```